tests/lib/rules/no-mixins.js

```diff
@@ -0,0 +1,66 @@
+// ------------------------------------------------------------------------------
+// Requirements
+// ------------------------------------------------------------------------------
+
+const rule = require('../../../lib/rules/no-mixins');
+const RuleTester = require('eslint').RuleTester;
+
+// ------------------------------------------------------------------------------
+// Tests
+// ------------------------------------------------------------------------------
+
+const { ERROR_MESSAGE } = rule;
+const eslintTester = new RuleTester({
+  parserOptions: { ecmaVersion: 6, sourceType: 'module' },
+});
+eslintTester.run('no-mixins', rule, {
+  valid: [
```

Comment:
Can you add `import Mixin from '@ember/object/mixin';` as a valid example? Note that the test cases only need to contain the import statement and nothing else.

Reply:
Yup, but I find that including more code usually catches more false positives.